backend/danswer/db/pg_file_store.py

```diff
@@ -131,3 +138,25 @@ def delete_pgfilestore_by_file_name(
     db_session: Session,
 ) -> None:
     db_session.query(PGFileStore).filter_by(file_name=file_name).delete()
+
+
+def delete_chat_files_older_than(days_old: int, db_session: Session) -> None:
```

Comment:
I would prefer to do this based on the `files` field in the `ChatMessage` table. This way we don't have to add any new columns + it feels more intuitive (e.g. we go through all old chat sessions, and then delete everything about each chat session including all messages, files, search docs, etc.)